core/src/main/scala/sttp/tapir/SchemaType.scala

```diff
@@ -73,6 +73,10 @@ object SchemaType {
   case class SObjectInfo(fullName: String, typeParameterShortNames: List[String] = Nil)
   object SObjectInfo {
     val Unit: SObjectInfo = SObjectInfo(fullName = "Unit")
+    val defaultToNameMapper: SObjectInfo => String = info => {
```

Comment:
this is only used for openapi/asyncapi generation, so I'd keep that code for use only in these modules. Maybe the package object `sttp.tapir.docs.apispec` would be a good place?

Reply:
corrected